server/src/main/java/org/opensearch/indices/NodeIndicesStats.java

```diff
@@ -281,7 +410,21 @@ public List<IndexShardStats> getShardStats(Index index) {
      *
      * @opensearch.internal
      */
-    static final class Fields {
-        static final String INDICES = "indices";
+    @PublicApi(since = "3.0.0")
```

Comment:
does it need to be Public?

Reply:
Enums I think are required to be defined @PublicApi. The pre-commit checks were failing without it.